ansible_tower_client.gemspec

```diff
@@ -19,10 +19,13 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  # HACK: Rails 5 dropped support for Ruby < 2.2.2
+  active_support_version = "< 5" if Gem::Version.new(RUBY_VERSION) <= Gem::Version.new("2.2.2")
+  spec.add_runtime_dependency "activesupport", active_support_version
```

Comment:
I agree, it would be nice to remove the dependency on `activesupport`, but right now we use a few very complicated parts of `activesupport/inflector`